fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -2695,26 +2695,33 @@ void manySegmentsParallelOpen(boolean primaryKeySegmentIndexEnabled) {
             assertEquals(20,
                     recordStore.scanIndex(index, fullTextSearch(index, "Vision"), null, ScanProperties.FORWARD_SCAN).getCount().join());
             try (FDBDirectory directory = new FDBDirectory(recordStore.indexSubspace(index), context, index.getOptions())) {
-                assertEquals(21, directory.listAll().length);
+                assertEquals(61, directory.listAll().length);
```

Comment:
Do you know why this went up with your PR?

Reply:
Hmm. This is the right value, though I don't know why it shows up as a diff. 
Must be something funny with the merge.
I'm going to try merging again as it shows conflicts anyway.

Reply:
IT changed on main, and showed up because of the merge.